bottlerocket/agents/src/bin/sonobuoy-test-agent/main.rs

```diff
@@ -82,9 +82,10 @@ impl test_agent::Runner for SonobuoyTestRunner {
                 .context(error::SecretMissingSnafu)?;
             setup_wireguard(&wireguard_secret).await?;
         }
-
-        decode_write_kubeconfig(&self.config.kubeconfig_base64, TEST_CLUSTER_KUBECONFIG_PATH)
+        info!("Decoding kubeconfig for test cluster");
```

Comment:
Nit:

Should these `info!` be `debug!`? Is this information useful for the user always, or only while debugging?